pkg/metrics/metrics.go

```diff
@@ -50,52 +53,88 @@ func InitPrometheus(config *config.Config) {
 
 	initialized = true
 
+	histogramLabels := make([]string, 0)
```

Comment:
can this become a variable and then just append to it.  Not asking for this, just a recommendation. See https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices where `[]string{}` is the same as doing `make([]string, 0)`.

Reply:
Okay, interesting. I can even do a range over nil that it causes an error.

Reply:
Done.